Etiquette/CODE_REVIEW.md

```diff
@@ -0,0 +1,108 @@
+
+Note: Guidelines in this document should be considered to be of equal importance
+
+Note: This document is heavily inspired by [this blog post](https://github.blog/2015-01-21-how-to-write-the-perfect-pull-request/)
+
+Everyone
+--------
+
+* Be respecful to each other
+* Be humble. ("I'm not sure - let's look it up.")
+* Don't use hyperbole. ("always", "never", "endlessly", "nothing")
+* Don't use sarcasm or trolling
+* If you feel annoyed at something/someone don't make it affect your or others work. Talk to a friend, go for a walk, meditate, switch to something else.
+* Talk synchronously, e.g. call or in person, prefer not to use chat for that, if there are too many "I didn't understand" or "Alternative solution:" comments. Post a follow-up
+  comment summarizing the discussion.
+* Avoid selective ownership of code ("mine", "not mine", "yours"). 
+* Use "we" instead of "you", unless asking for opinion of specific individual ("What do you think ...?", but "Here we are doing ... What if we ...?",)
+* Avoid using terms that could be seen as referring to personal traits. ("dumb",
+  "stupid", "insane"). Assume everyone is intelligent and well-meaning.
+* Be aware of negative bias with online communication. (If content is neutral, we assume the tone is negative.) Can you use positive language as opposed to neutral?
+* When using emojis be aware that their meaning can be different for different individuals - winking smile may not be always innosent
+* Same with any other media format you decide to use
+* Do not expect your comments to be answered immediately. If you need an answer fast - talk in person (same applies to the Slack messages)
+* It's fine to have disagreements and its hard to please everyone.
+
+
+Creating Pull Request
+--------
+
+* Carefully fill in pull request template:
+	* title of the PR should follow format `[XXX-123] Description of the change` starting with a ticket number
+	* explain the context and motivation, don’t assume familiarity with the history. Highlight how the goal was achived without going too much into details - people tend not to read long descriptions and if the long explanation is needed probably just pull request will be not enough (the change may need to go through proposal process). 
+	* don't strip out parts of the template unless it is absolutely unrelevant to the changes (i.e. change in the build script does not need any screenshots as it does not touch UI)
+	* don't forget labels (it's better to set them after PR is created as github sometimes fails with 405 status code and we don't know why)
+* explicitly request review from or mention team mates you specifically want to involve in the discussion
+* Self-review your changes, adding comments where you think you can get questions (it may worth adding these as code or documentation comments). You can use github suggestions and apply them right away
```

Comment:
I agree with the idea (that GitHub suggestions should be used when possible), but don't really get what this is mentioned here in relation to this "self-review your code" point? At least I didn't understand what this had to do with anything when reading this point 😅 

It's rightfully already mentioned in the next section ("Review Pull Requests") and has its place down there, so I think adding it here too adds confusion more than anything.

```suggestion
* Self-review your changes, adding comments where you think you can get questions (it may worth adding these as code or documentation comments)
```

Reply:
I mean that if when doing self-review one spots small issues they can be addressed with GitHub suggestions.

Reply:
ah got it, keep it then 👍